app/components/preprint-status-banner/component.js

```diff
@@ -227,11 +229,12 @@ export default Component.extend({
         return this.get('submission.reviewsState') !== this.get('decision');
     }),
 
+    notDefaultValues: computed('submission.reviewsState', 'decision', 'commentEdited', function() {
+        return this.get('decision') !== ACCEPTED || this.get('commentEdited');
```

Comment:
For rejected submissions, the "default" is `REJECTED`, so this doesn't prompt the user if they try to leave after changing from rejected to accepted. It might be good to save an `initialDecision` somewhere, which is the initial selected value.